src/please.go

```diff
@@ -614,7 +614,7 @@ var buildFunctions = map[string]func() int{
 		return toExitCode(help.Help(string(opts.Help.Args.Topic)), nil)
 	},
 	"tool": func() int {
-		tool.Run(config, opts.Tool.Args.Tool, opts.Tool.Args.Args.AsStrings())
+		runTool(opts.Tool.Args.Tool)
```

Comment:
can we make this `return runTool(...)` and return a non-zero exit code when the build fails?